media-types.md

```diff
@@ -47,3 +47,5 @@ The following figure shows how the above media types reference each other:
 
 [Descriptors](descriptor.md) are used for all references.
 The manifest list being a "fat manifest" references one or more image manifests per target platform. An image manifest references exactly one target configuration and possibly many layers.
+
+[JSON]: http://json.org/
```

Comment:
[First line in the JSON section][1] ;).

[1]: https://github.com/opencontainers/image-spec/pull/339/files#diff-a40e69a354ae90a6066b6786869dc9dbR12